### DISCUSSION TITLE
[@types/ember]

### DISCUSSION THREAD

**Discussion by kas1337** [DefinitelyTyped/DefinitelyTyped#67096] [Issues with a @types package] [answered]
Since ~~v3.16.11~~ v4.0.7 `ember/-private/type-utils` containing type declarations is missing. Could you please fix this? Thank you!

[a comment by an automated account is not shown]

**Comment by jakebailey** [comment]
There's no -private in the v3 dir at all, though.

**Reply by jakebailey** [reply]
And looking at the old published package, it's not there either.

**Reply by jakebailey** [reply]
https://unpkg.com/browse/@types/ember@3.16.10/

https://unpkg.com/browse/@types/ember@3.16.11/

https://app.renovatebot.com/package-diff?name=@types%2fember&from=3.16.10&to=3.16.11

I see no unexpected changes.

**Reply by kas1337** [reply]
My bad. I looked up the wrong versions. -private is missing in v4.0.7. It was there in v4.0.6.

**Reply by kas1337** [reply]
I updated the description.

**Reply by jakebailey** [reply]
This is a bug in the publisher; I am working on fix.

**Reply by jakebailey** [reply, marked as answer]
Should be fixed now for ember. The fix: https://app.renovatebot.com/package-diff?name=@types%2fember&from=4.0.7&to=4.0.8

The _intended_ change post-pnpm https://app.renovatebot.com/package-diff?name=@types%2fember&from=4.0.6&to=4.0.8
